bond/bond.py

```diff
@@ -235,12 +265,9 @@ def change_filename(self, filepath, entities):
         else:
             print("FOUND IRREGULAR NUMBER OF JSONS")
 
-    def fieldmaps_ok(self):
-        pass
-
     def _cache_fieldmaps(self):
-        """Searches all fieldmaps and creates a lookup for each file.
-        """
+        '''Searches all fieldmaps and creates a lookup for each file.'''
```

Comment:
This function returns a list of misfits. Does that get used elsewhere? if we want to keep it, we should add it to the docstring or we should remove it. A function called _cache suggests that the returned object should be some sort of cache or None

Reply:
I think the misfits are something Ted wanted to keep track of, so I think we should keep the list as a return. I will update the docstring.